### PR TITLE
Added contextual help links

### PR DESCRIPTION
**Pull request by mpaglia0**

### CODE REVIEW

**Comment by mpaglia0** [comment]
I will now stop waiting your approval for the changes done till now.
Thanks

File: src/libs/collect.c [hunk 1]
```diff
@@ -1391,17 +1392,20 @@ static void combo_changed(GtkComboBox *combo, dt_lib_collect_rule_t *d)
      || property == DT_COLLECTION_PROP_ISO)
   {
     gtk_widget_set_tooltip_text(d->text, _("type your query, use <, <=, >, >=, <>, =, [;] as operators"));
+    dt_gui_add_help_link(d->text, "collect_images.html#collect_images_usage");
```
Comment (houz) [inline src/libs/collect.c]: You don't need to set the help text in this function, just do it once in `gui_init()` when the widgets are created.

File: src/libs/colorpicker.c [hunk 2]
```diff
@@ -142,6 +142,7 @@ static void _update_picker_output(dt_lib_module_t *self)
     darktable.gui->reset = 1;
     gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(data->picker_button),
                                  module->request_color_pick != DT_REQUEST_COLORPICK_OFF);
+    dt_gui_add_help_link(data->picker_button, "global_color_picker.html#global_color_picker");
```
Comment (houz) [inline src/libs/colorpicker.c]: Again, only set the help text once when creating a widget.

**Comment by TurboGit** [comment]
Indeed, thanks for the reminder.

**Pull request closed without merging**